<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserEdit</class>
 <widget class="QWidget" name="UserEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Отредактировать</string>
  </property>
  <widget class="QPushButton" name="pushButtonUserEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>360</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFirstNameEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>55</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLastNameEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Фамилия</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowserUserEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>311</width>
     <height>401</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; text-decoration: underline;&quot;&gt;Редактирование пользователя&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNumberPhoneEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Номер телефона</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelIdUserEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>55</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditFLastNameEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditNumberPhoneEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditFirstNameEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditIdUserEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <zorder>textBrowserUserEdit</zorder>
  <zorder>pushButtonUserEdit</zorder>
  <zorder>labelFirstNameEdit</zorder>
  <zorder>labelLastNameEdit</zorder>
  <zorder>labelNumberPhoneEdit</zorder>
  <zorder>labelIdUserEdit</zorder>
  <zorder>lineEditFLastNameEdit</zorder>
  <zorder>lineEditNumberPhoneEdit</zorder>
  <zorder>lineEditFirstNameEdit</zorder>
  <zorder>lineEditIdUserEdit</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
